test/webserv/integration/common.py

```
@@ -0,0 +1,90 @@
+
+response_header_get_root_200_close = create_response_header(
+    200, "OK", CLOSE, root_index_file_length
+)
+response_header_get_root_200_keep = create_response_header(
+    200, "OK", KEEP_ALIVE, root_index_file_length
+)
+response_header_get_sub_200_close = create_response_header(
+    200, "OK", CLOSE, sub_index_file_length
+)
+response_header_400 = create_response_header(
+    400, "Bad Request", CLOSE, bad_request_file_400_length
```

Ruff

Undefined name `bad_request_file_400_length`

(F821)